api/jobs/handlers.py

```diff
@@ -328,40 +280,30 @@ def next(self):
         else:
             return job
 
+    @require_admin
     def reap_stale(self):
-        if not self.superuser_request and not self.user_is_admin:
-            self.abort(403, 'Request requires admin')
-
         count = Queue.scan_for_orphans()
         return { 'orphaned': count }
 
 class JobHandler(base.RequestHandler):
     """Provides /Jobs/<jid> routes."""
 
+    @require_admin
     def get(self, _id):
-
-        if not self.superuser_request and not self.user_is_admin:
-            self.abort(403, 'Request requires admin')
-
         return Job.get(_id)
 
+    @require_admin
     def get_config(self, _id):
         """Get a job's config"""
-        if not self.superuser_request and not self.user_is_admin:
-            self.abort(403, 'Request requires superuser')
-
         j = Job.get(_id)
         c = j.config
         if c is None:
             c = {}
 
-        # Serve config as formatted json file
-        self.response.headers['Content-Type'] = 'application/octet-stream'
-        self.response.headers['Content-Disposition'] = 'attachment; filename="config.json"'
-
         # Detect if config is old- or new-style.
         # TODO: remove this logic with a DB upgrade, ref database.py's reserved upgrade section.
```

Comment:
I believe this db upgrade has been completed, can we remove the check below?

Reply:
I don't know. The instructions in both locations were very clear, so I'm assuming not.